### ISSUE TITLE
Filtered format (table view): The each() function is deprecated.

### ISSUE THREAD

**Issue by kghbln** [SemanticMediaWiki/SemanticResultFormats#478]
### Setup

- MediaWiki 	1.31.1 (fe7ffa2) 21. Mär. 2019, 19:54
- PHP 	7.2.16 (cgi-fcgi)
- MariaDB 	10.1.38-MariaDB
- Semantic MediaWiki | 3.0.1
- Semantic Result Formats | 3.0.0 (14e04f1) 22. Feb. 2019, 20:51

### Issue
```
Deprecated: The each() function is deprecated. This message will be suppressed on further calls in /.../extensions/SemanticResultFormats/formats/filtered/src/View/TableView.php on line 94
```

**Comment by kghbln** [comment]
@s7eph4n FYI

**Comment by kghbln** [comment]
@s7eph4n It will be cool to have a fix for SRF 3.1.0 for this one.
